src/Main.cc

```diff
@@ -412,7 +423,7 @@ static int tclAppInit(int& argc,
       init_path += init_filename;
       if (is_regular_file(init_path.c_str())) {
         if (!gui_enabled) {
-          sourceTclFile(init_path.c_str(), true, true, interp);
+          tclEvalFile(init_path.c_str(), interp);
```

Comment:
sourceTclFile provides some error checking etc and feedback. It feels odd to me that if I run source I get some kind of feedback but if I run openroad with a tcl script I dont

Reply:
you get the same errors and feedback as you would get if you ruh "tslsh foo.tcl"

Reply:
But not those that match `source`. It feels like the correct solution would be to wait for opensta to implement this so it's consistent throughout openroad

Reply:
I think OpenSTA is a red herring here. This is about how OpenROAD chooses to use Tcl, it has nothing to do with OpenSTA.

Here is the OpenSTA function, what would you have it do differently?

https://github.com/The-OpenROAD-Project/OpenSTA/blob/1c7f022cd0a02ce71d047aa3dbb64e924b6efbd5/app/StaMain.cc#L84C1-L100C1

Reply:
Actually this is the function https://github.com/parallaxsw/OpenSTA/blob/6110a6b96563676228bd51f95a21ae0a2da2aa92/tcl/Sdc.tcl#L68

Reply:
Aha! That was surprising. Any idea why "source" has been overriden instead of keeping an OpenSTA specific name?

Reply:
I suspect to provide better error messaging than what is available in tcl. Because of this, i think the right solution is to have the opensta issue resolved, which should solve this globally for us

Reply:
This is the tcl behavior of "source":

```
$ cat blah.tcl 
puts "Script running: [info script]"
$ tclsh
% source blah.tcl
Script running: blah.tcl
$ tclsh 
% source ~/OpenROAD-flow-scripts/blah.tcl
Script running: ~/OpenROAD-flow-scripts/blah.tcl
$ tclsh 
% source ~/OpenROAD-flow-scripts/blah.tcl
Script running: ~/OpenROAD-flow-scripts/blah.tcl
% source OpenROAD-flow-scripts/blah.tcl
Script running: OpenROAD-flow-scripts/blah.tcl
```

From inside `~/OpenROAD-flow-scripts`:

```
$ tclsh
% source ../OpenROAD-flow-scripts/blah.tcl
Script running: ../OpenROAD-flow-scripts/blah.tcl
```

Reply:
> I suspect to provide better error messaging than what is available in tcl. Because of this, i think the right solution is to have the opensta issue resolved, which should solve this globally for us

That's still possible without replacing the existing `source` function. I wonder why the default tcl behavior was overriden instead of having a separate sta_source command.